packages/artifacts/src/addresses.json

```diff
@@ -0,0 +1,98 @@
+{
+    "dao": {
+        "mainnet": "",
+        "sepolia": "",
+        "holesky": "",
+        "devSepolia": "",
+        "polygon": "",
+        "mumbai": "",
+        "base": "",
+        "baseSepolia": "",
+        "arbitrum": "",
+        "arbitrumSepolia": "",
+        "linea": "",
+        "lineaSepolia": "",
+        "zksync": "",
+        "zksyncSepolia": ""
+    },
+    "daoFactory": {
+        "mainnet": "",
+        "sepolia": "",
+        "holesky": "",
+        "devSepolia": "",
+        "polygon": "",
+        "mumbai": "",
+        "base": "",
+        "baseSepolia": "",
+        "arbitrum": "",
+        "arbitrumSepolia": "",
+        "linea": "",
+        "lineaSepolia": "",
+        "zksync": "",
+        "zksyncSepolia": ""
+    },
+    "daoRegistry": {
+        "mainnet": "",
+        "sepolia": "",
+        "holesky": "",
+        "devSepolia": "",
+        "polygon": "",
+        "mumbai": "",
+        "base": "",
+        "baseSepolia": "",
+        "arbitrum": "",
+        "arbitrumSepolia": "",
+        "linea": "",
+        "lineaSepolia": "",
+        "zksync": "",
+        "zksyncSepolia": ""
+    },
+    "pluginRepoFactory": {
+        "mainnet": "",
+        "sepolia": "",
+        "holesky": "",
+        "devSepolia": "",
+        "polygon": "",
+        "mumbai": "",
+        "base": "",
+        "baseSepolia": "",
+        "arbitrum": "",
+        "arbitrumSepolia": "",
+        "linea": "",
+        "lineaSepolia": "",
+        "zksync": "",
+        "zksyncSepolia": ""
+    },
+    "pluginRepoRegistry": {
+        "mainnet": "",
+        "sepolia": "",
+        "holesky": "",
+        "devSepolia": "",
+        "polygon": "",
+        "mumbai": "",
+        "base": "",
+        "baseSepolia": "",
+        "arbitrum": "",
+        "arbitrumSepolia": "",
+        "linea": "",
+        "lineaSepolia": "",
+        "zksync": "",
+        "zksyncSepolia": ""
+    },
+    "pluginSetupProcessor": {
+        "mainnet": "",
+        "sepolia": "",
+        "holesky": "",
+        "devSepolia": "",
+        "polygon": "",
+        "mumbai": "",
+        "base": "",
+        "baseSepolia": "",
+        "arbitrum": "",
+        "arbitrumSepolia": "",
+        "linea": "",
+        "lineaSepolia": "",
+        "zksync": "",
+        "zksyncSepolia": ""
+    }
```

Comment:
add addresses for xyzRegistry and psp as we know those won't change

Reply:
👍

Reply:
no, it will stay, verify it with the deploy script